boost-maven/boost-maven-plugin/src/it/test-liberty-dev-mode/src/test/java/net/wasdev/wlp/test/dev/it/DevHotTestingTest.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * (c) Copyright IBM Corporation 2019.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package net.wasdev.wlp.test.dev.it;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DevHotTestingTest extends BaseDevTest {
```

Comment:
Do you think there's value in testing the "hotTests" path in boost specifically?   If not maybe we could simplify by collapsing this into one class and reducing one moving part.